packages/hawtio/src/ui/notification/HawtioNotification.tsx

```diff
@@ -36,11 +36,11 @@ export const HawtioNotification: React.FunctionComponent = () => {
     const listener = (notification: Notification) => {
       const key = getUniqueKey()
       addAlert(notification.message, notification.type, key)
-      if (notification.duration) {
-        setTimeout(() => {
-          removeAlert(key)
-        }, notification.duration)
-      }
+      // if (notification.duration) {
+      //   setTimeout(() => {
+      //     //    removeAlert(key)
+      //   }, notification.duration)
+      // }
```

Comment:
Why did you comment this out?

Reply:
Thanks! I was playing with notifications and accidently commited it. Should be gone now.